crates/vault-registry/src/lib.rs

```diff
@@ -142,6 +148,9 @@ decl_storage! {
 
         /// Build storage at V1 (requires default 0).
         StorageVersion get(fn storage_version) build(|_| Version::V1): Version = Version::V0;
+
+        /// Map of Vault Operators
+        IsNominationOperator: map hasher(blake2_128_concat) T::AccountId => bool;
```

Comment:
The idea for this feature is to cause as few changes as possible to the current tests. Including this map in the `vault` struct would cause changes in all the mocked objects

Reply:
Why do we track this in the vault-registry? Would it be possible to move into the nomination pallet?